Backend/src/main/java/idealab/api/operations/UserOperations.java

```diff
@@ -34,4 +34,25 @@ public GenericResponse userSignUp(Employee login) {
         response.setMessage("User Sign Up Successful");
         return response;
     }
+
+    public GenericResponse deleteUser(Integer id) {
+        GenericResponse response = new GenericResponse();
+        Employee e = employeeRepo.findEmployeeById(id);
+
+        if(e != null) {
+            try {
+                employeeRepo.deleteById(id);
+                response.setSuccess(true);
+                response.setMessage("Employee Deleted Successfully");
+            } catch (Exception ex) {
```

Comment:
I think we should be logging this exception.

Reply:
I'm not totally sure what we're doing as far as logging yet. There's some logging in the controllers, but for the operations I've printed the stacktrace. We can make an issue to revisit logging later.